app/src/test/java/org/simple/clinic/recentpatient/AllRecentPatientsLogicTest.kt

```diff
@@ -161,8 +162,8 @@ class AllRecentPatientsLogicTest {
         clock = userClock,
         isNewRegistration = true
     )))
-    verify(ui).openPatientSummary(patientUuid)
-    verifyNoMoreInteractions(ui)
+    verify(uiActions).openPatientSummary(patientUuid)
+    verifyNoMoreInteractions(ui, uiActions)
```

Comment:
Should we also add `verifyZeroIneractions(uiActions)` in the other test?

Reply:
Yes, good catch!